### PR TITLE
Unify the /community and /contact pages

### LINKED ISSUE #917: Improve the contact page
The [/contact page](https://electronjs.org/contact) currently lists a number of ways to get in touch, but does not clearly define what people should expect from each channel, or how they should interact with it:

- GitHub @electron
- Twitter @electronjs (we pretty much never respond to replies using @electronjs, in favor of our personal accounts)
- Slack atomio.slack.com (not frequented by most Electron team members)
- Discuss discuss.atom.io/c/electron (also not frequented by team members)
- Email electron@github.com (currently a dumping ground, mostly spam)

Let's make the page more actionable, like:

- To report a bug or request a feature for Electron, open an issue in electron/electron
- To report a security incident, email security@electronjs.org
- To report a violation of the Code of Conduct, email coc@electronjs.org
- To add your app to the site, open a PR on the electron-apps repo
- etc..

### PR DESCRIPTION
**Pull request by zeke**
This PR:

- adds instructions to the top of the /community page for: staying up-to-date, getting help, reporting bugs, making feature requests, reporting CoC violations, reporting security vulnerabilities, etc 💡 
- removes the old `/contact` route and redirects to `/community` 👉 
- removes Contact link from header and footer. (Community is still there) 🔗 
- fixes a bug in our English-backfilling localization magic 🎩 
- add a list of non-English language resources 🌍 
- confirms that our tests around locale keys and templates are working 💪 
- is localized, so it will eventually get translated by our translator community. 💛 

Thanks for @ckerr @jkleinsc @nornagon for the brainstorm session this morning.

Review app: https://electron-website-pr-1234.herokuapp.com/community

Resolves #917

### CODE REVIEW

File: views/community.html [hunk 1]
```diff
@@ -7,8 +7,33 @@ <h1>{{localized.community.title}}</h1>
 
 <section class='page-section'>
 
-  <div class="container-narrow text-center">
+  <div class="container-narrow">
+    <p class="lead mb-4">📣 {{{localized.community.channels.updates}}}</p>
+    <p class="lead mb-4">🙋 {{{localized.community.channels.help}}}</p>
+    <p class="lead mb-4">🐞 {{{localized.community.channels.bugs}}}</p>
+    <p class="lead mb-4">💡 {{{localized.community.channels.feature_requests}}}</p>
+    <p class="lead mb-4">👎 {{{localized.community.channels.code_of_conduct}}}</p>
```
[the remark this answers is not in the document]
Comment (zeke) [reply views/community.html]: Yeah good call. What should we use instead?

[the remark this answers is not in the document]
Comment (zeke) [reply views/community.html]: 👋 👨‍👩‍👧‍👦 🤝 ⚖️

[the remark this answers is not in the document]
Comment (vanessayuenn) [reply views/community.html]: I like 🤝!!

[the remark this answers is not in the document]
Comment (zeke) [reply views/community.html]: ✅

**Comment by zeke** [comment]
image

- [x] added a link to the languages page
- [x] added a link to the package search page
- [x] made everything a call to action

**Comment by groundwater** [comment]
Request: Add _Contact_ link in the footer, it can point to `/community`.
Reason: If an outside party enters electronjs.org to disclosure a security vulnerability or other sensitive issue, I think _Contact_ is the first thing they would search for.

**Comment by zeke** [comment]
https://twitter.com/electronjs/status/991400931685105664

**Pull request merged**